docs/en/Security.md

```diff
@@ -126,6 +127,58 @@ The owner, group, and permissions can be changed by two ways:
 The owner can only be changed by super user.
 The group and permission can only be changed by super user and file owner.
 
+## Access Control Lists
+Posix permission model allows administrators to grant permissions to owners, owning groups and other users. 
+The permission bits model is sufficient for most cases. 
+However, to help administrators express more complicated security policies,
+Alluxio also supports Access Control Lists (ACLs).
+ACLs allow administrators to grant permissions to any user or group. 
+
+In Alluxio's ACL model, a file or directory's Access Control List consists of many ACL entries. There are two types of ACL entries, Access ACL entries and Default ACL entries. 
+
+1. Access ACL Entries:
+
+   This type of ACL entry specifies a particular user or group's permission to read, write and execute. 
```

Comment:
nit: I'd leave out the start of paragraph indentation to be consistent with our other docs. I think markdown ignores it anyway

Reply:
it does matter i think. This way the word `this` is aligned with `Access` , otherwise it starts at the beginning of the line.

Reply:
sgtm